### ISSUE TITLE
[REQUEST] - Auto roll on added roll tables

### ISSUE THREAD

**Issue by Axblade** [fantasycalendar/FoundryVTT-ItemPiles#377]
**Is your feature request related to a problem? Please describe.**
No there's no problem overall just quality of life.

**Describe the solution you'd like**
Enable a option when token that is set as merchant or make addition pile type as a shelf or counter. That when place item into a scene it will roll the added roll tables with quantities set.

**Describe alternatives you've considered**
I considered that maybe doing this as a macro maybe a solution however uncertain what the commands would be.

**Additional context**
Basically imagen you have a weapon rack and when place in a scene want it roll the table see what swords are on the rack

**Comment by Haxxer** [comment]
So in short, you want to be able to roll on a table to add items to an item pile. How are shelves or counters different than a normal item pile?

**Comment by Axblade** [comment]
Maybe I see in macro you have set the table and dice. In item pile settings you can add those but to populate have click the dice to roll it and add button to add it to inventory. If there was feature to do those automatically upon being created in scene that be great. Maybe adjusting macro or adding a function/variable to pull tables and amounts not sure if those already exist. Still learning macros for foundry so sorry if just missing it.
